### DISCUSSION TITLE
Shop

### DISCUSSION THREAD

**Discussion by Gunstra** [Anduin2017/AnduinOS#27] [Q&A] [answered]
Work they're be a shop eventually instead of a website shop? Like Discover or something? Easier for downloading and updating. Especially if you want users to get the windows 11 feel

**Comment by Anduin2017** [comment]
Yes, I have thought about this as well. However, I also believe that traditional package management systems each have their shortcomings: Nix, with its functional modules, is difficult for the average user to grasp, while Arch's AUR is often considered too messy and hard to control. Ubuntu and Debian's apt is good, but once a package conflict arises, it can be extremely challenging to resolve. The current mainstream industry solutions are snap, AppImage, and Flatpak. I think it might be a good idea to continue using apt to manage system components while using Flatpak for user-installable graphical apps. Additionally, I have recently been experimenting with some interesting technologies, such as deploying graphical software directly through Docker.

**Comment by Anduin2017** [comment]
Additionally, I did not deliberately make users feel that this system is imitating Windows 11. In fact, I believe there are some aspects of Windows 11's design that are worth learning from, but there is also a lot of undesirable baggage. I dislike the excessive ads and pervasive AI features in Windows 11, as well as how many processes, such as automatic updates, are made opaque and hidden from view. I still want to have control over my own computer, which is why I chose Linux.

**Comment by Anduin2017** [comment, marked as answer]
Additionally, many software license agreements explicitly state that unauthorized distribution is illegal. If we operate our own app store, such software that we lack the rights to distribute cannot be included in the store. We cannot take on this legal risk. For example: Google Chrome, Skype, Zoom, WhatsApp, ElasticSearch, Minecraft, Steam, Epic Store, EA Origin, etc. Therefore, the best option we can provide is to guide users on how to install these applications via their official channels on Linux and organize one-click installation methods through the official channels.
